tests/Oryx.Integration.Tests/LocalDockerTests/Fixtures/DbContainerFixtureBase.cs

```diff
@@ -31,11 +30,32 @@ public abstract class DbContainerFixtureBase : IDisposable
         public DbContainerFixtureBase()
         {
             DbServerContainerName = RunDbServerContainer().ContainerName;
-            if (!WaitUntilDbServerIsUp())
+
+            bool insertedSampleData = false;
+            try
+            {
+                var isDbServerUp = WaitUntilDbServerIsUp();
+                if (isDbServerUp)
+                {
+                    InsertSampleData();
+                    insertedSampleData = true;
+                }
+            }
+            catch (Exception ex)
             {
-                throw new Exception("Database not ready in time");
+                // Since Dispose method does not get called in case of exceptions, catch the exception here, log it
```

Comment:
This was one of the issues that was happening earlier where dispose wasn't get called (but yeah that does not explain why stopping the containers fails in our uber clean-up script)